docs/argument_processing.rst

```diff
@@ -214,7 +214,13 @@ Sub-commands
 Sub-commands are supported for commands using either the ``@with_argparser`` or
 ``@with_argparser_and_unknown_args`` decorator.  The syntax for supporting them is based on argparse sub-parsers.
 
+Also, a convenience function called ``cmd_with_subs_completer`` is available to easily add tab completion to functions
```

Comment:
Thank you for coming back and adding documentation and unit tests for your previous PR!